src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -489,7 +489,7 @@ function showVersionWarningBanner(data) {
   middle.classList = "bd-header-announcement__content";
   inner.classList = "sidebar-message";
   button.classList =
-    "sd-btn sd-btn-danger sd-shadow-sm sd-text-wrap font-weight-bold ms-3 my-1 align-baseline";
+    "btn text-wrap font-weight-bold ms-3 my-1 align-baseline button-link-to-stable-version";
```

Comment:
```suggestion
    "btn text-wrap font-weight-bold ms-3 my-1 align-baseline pst-button-link-to-stable-version";
```
let's follow the convention to use `pst-` for our own styles/components

Reply:
Good catch!